docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -13,6 +13,7 @@ Block themes require WordPress 5.9. To use block themes in earlier versions of W
 3.  [Creating the templates and template parts](#creating-the-templates-and-template-parts)
 4.  [Theme.json - Global styles](#themejson---global-styles)
 5.  [Custom templates](#custom-templates)
+6.  [Global styles presets](#global-styles-presets)
```

Comment:
Nit: this is shown properly in the `md` but we could correct the markup as well maybe..
I'm referring to the numbered items (5, 6, 6).